server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -313,7 +313,8 @@ public CompactionStats call() throws IOException, CompactionCanceledException {
 
         reader = fileFactory.newReaderBuilder()
             .forFile(mapFile.getPathStr(), fs, fs.getConf(), cryptoService)
-            .withTableConfiguration(acuTableConf).withRateLimiter(env.getReadLimiter()).build();
+            .withTableConfiguration(acuTableConf).withRateLimiter(env.getReadLimiter())
+            .dropCachesBehind().build();
```

Comment:
@dlmarion  this is setting the compaction input files to drop behind.  Did you consider setting it for the compaction output file as well?

Reply:
> this is setting the compaction input files to drop behind

Yes

> Did you consider setting it for the compaction output file as well?

I put my thoughts [here](https://github.com/apache/accumulo/issues/3078#issuecomment-1315293006). Maybe you know the answer...

Reply:
I just saw that comment,  I didn't see it before asking here.